tests/lib/rules/no-implicit-coercion.js

```diff
@@ -28,6 +28,11 @@ ruleTester.run("no-implicit-coercion", rule, {
         {code: "~foo"},
         {code: "-foo"},
         {code: "2 * foo"},
+        {code: "1 * 1234"},
```

Comment:
I want to warn this pattern with a message like `This "1*" is no effect.`...
But I'm not sure it should do in this rule.